renovate.json

```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"extends": [
+		"config:base",
+		"schedule:weekdays",
+		":automergeRequireAllStatusChecks"
+	],
+	"baseBranches": [
+		"develop"
+	],
+	"commitMessagePrefix": "[TASK] ",
+	"commitMessageTopic": "{{depName}}",
+	"force": {
+		"constraints": {
+			"php": "8.1.*"
```

Comment:
Why do we need the explicit PHP constraint? Renovate should be able to extract it from the `composer.json`.

Reply:
Ah, that was me, sorry 🤔 Don't know why I added this. Will have a look at it.

Reply:
I probably set this to 8.1 because in CI we still use PHP 8.1 because of PHP-CS-Fixer not yet supporting PHP 8.2.

I changed the CGL workflow to use PHP 8.2 in 98463a1 and removed the constraint here. This way, Renovate will run with PHP 8.2. The PHP-CS-Fixer job now ignores that an unsupported PHP version is used. We should have an eye on that, that's why I added a TODO to the appropriate configuration in the CGL workflow.